### PR TITLE
[Good to merge]Updates the Eris Map for Crew Monitoring and Security Monitoring

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Upgrades the Eris Map for the apps mentioned
eris-1
eris-2
eris-3
eris-4
eris-5
 (And no , we don't have a inbuilt tool that does this , i looked)

## Why It's Good For The Game
The current maps are severely outdated , and also too small.

## Changelog
:cl:
add: New map images for crew monitoring and security monitoring map mode
tweak: Added Icon32 for the crew monitor.
/:cl:

### CODE REVIEW

File: nano/templates/crew_monitor.tmpl [hunk 1]
```diff
@@ -72,7 +72,7 @@ Used In File(s): code\modules\modular_computers\file_system\programs\medical\sui
 				<td><span class="white">Poisoning:</span> Not Available</td>
 			{{/if}}
 			{{if value.sensor_type >= 3}}
-				<td><span class="white">Coordinates:</span> {{:value.x}}, {{:value.y}}, {{:value.z}}</td>
+				<td><span class="white">Coordinates:</span> {{:value.x / 1.285}}, {{:value.y / 1.285}}, {{:value.z}}</td>
```
Comment (TheShown911) [inline nano/templates/crew_monitor.tmpl:75]: These values should be passed as data from the define in `code/datums/repositories/crew/tracking.dm`

File: nano/templates/crew_monitor_map_content.tmpl [hunk 2]
```diff
@@ -6,7 +6,7 @@ Used In File(s): code\modules\modular_computers\file_system\programs\medical\sui
 	{{if value.sensor_type == 3 && value.z == config.mapZLevel}}
 		<div class="mapIcon mapIcon16 rank-{{:value.rank.ckey()}} {{:value.alert ? 'dead' : 'alive'}}" style="left: {{:(value.x + 0.2)}}px; bottom: {{:(value.y - 14.75)}}px;" unselectable="on">
 			<div class="tooltip hidden">
-				{{:value.name}} ({{:value.assignment}}) - {{:value.alert ? 'Medical issue detected' : 'No issues detected'}} - {{:value.area}} ({{:value.x}}, {{:value.y}})
+				{{:value.name}} ({{:value.assignment}}) - {{:value.alert ? 'Medical issue detected' : 'No issues detected'}} - {{:value.area}} ({{:value.x / 1.285}}, {{:value.y / 1.285}})
```
Comment (TheShown911) [inline nano/templates/crew_monitor_map_content.tmpl:9]: Here too.

File: nano/templates/sec_camera_map_content.tmpl [hunk 3]
```diff
@@ -4,7 +4,7 @@ Used In File(s): \code\game\machinery\computer\camera.dm
  -->
 {{for data.cameras}}
 	{{if value.z == config.mapZLevel}}
-		<div class="mapIcon mapIcon16" style="left: {{:(value.x)}}px; bottom: {{:(value.y - 14.75)}}px;">
+		<div class="mapIcon mapIcon16" style="left: {{:(value.x * 1.285 )}}px; bottom: {{:(value.y * 1.285 - 14.75)}}px;">
```
Comment (TheShown911) [inline nano/templates/sec_camera_map_content.tmpl:7]: Here too.

**Comment by hyperioo** [comment]
Any reason to revert #7399 ?

**Comment by MLGTASTICa** [comment]
> Any reason to revert #7399 ?

so it doesn't conflict. it won't revert on main.

**Pull request merged**